src/WebUI/Components/Common.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from "react";
+import { BaseComponent } from "@uifabric/utilities/lib";
+import { IStatusProps, Status, StatusSize } from "azure-devops-ui/Status";
+import { IVssComponentProperties } from "../Types";
+
+export interface IPodStatusProps extends IVssComponentProperties {
+    statusProps?: IStatusProps;
+    statusSize: StatusSize;

```

Comment:
status size as  statusSize={StatusSize.m}  and make it optional

Reply:
Done